streampark-console/streampark-console-service/src/test/java/org/apache/org/apache/streampark/common/util/CompletableFutureUtilsTest.java

```diff
@@ -143,4 +150,73 @@ private String runStart(int sec) {
     }
     return "start successful";
   }
+
+  @Test
+  public void thenSupplyNormally() throws Exception {
+    String successResult = "success";
+    String exceptionResult = "error";
+
+    String resp =
+        CompletableFutureUtils.supplyTimeout(
+                CompletableFuture.supplyAsync(() -> successResult),
+                1,
+                TimeUnit.SECONDS,
+                success -> success,
+                e -> exceptionResult)
+            .thenApply(r -> r)
+            .get();
+
+    Assertions.assertEquals(resp, successResult);
+  }
+
+  @Test
+  public void thenSupplyTimeout() throws Exception {
+    String successResult = "success";
+    String exceptionResult = "error";
+    CompletableFuture<String> future =
+        CompletableFuture.supplyAsync(
+            () -> {
+              try {
+                Thread.sleep(5000);
+              } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+              }
+              return successResult;
+            });
+    String resp =
+        CompletableFutureUtils.supplyTimeout(
+                future, 1, TimeUnit.SECONDS, success -> success, e -> exceptionResult)
+            .thenApply(r -> r)
+            .get();
+    Assertions.assertEquals(resp, exceptionResult);
+  }
+
+  @Test
+  public void thenSupplyException() {
+    String expectedExceptionMessage = "Exception in exceptionally handler";
+    int processTime = 5000;
+    int timeOut = 1000;
+    CompletableFuture<String> future =
+        CompletableFutureUtils.supplyTimeout(
+            CompletableFuture.supplyAsync(
+                () -> {
+                  try {
+                    Thread.sleep(processTime);
+                  } catch (InterruptedException e) {
+                    throw new RuntimeException(e);
+                  }
+                  return "success";
+                }),
+            timeOut,
+            TimeUnit.MILLISECONDS,
+            success -> success,
+            e -> {
+              throw new RuntimeException(expectedExceptionMessage);
+            });
+
+    assertThatThrownBy(future::get)
+        .hasMessageContaining(expectedExceptionMessage)
+        .hasCauseInstanceOf(RuntimeException.class)
+        .isInstanceOf(ExecutionException.class);
```

Comment:
```suggestion
    assertThatThrownBy(future::get)
        .cause()
        .as(expectedExceptionMessage)
        .isInstanceOf(RuntimeException.class);
```

Reply:
> 

Cannot resolve method 'cause' in 'AbstractThrowableAssert'

Reply:
> .isInstanceOf(RuntimeException.class);

 assertThatThrownBy(future::get)
        .getCause()
        .as(expectedExceptionMessage)
        .isInstanceOf(RuntimeException.class);

Reply:
done